src/main/java/org/dependencytrack/model/Component.java

```diff
@@ -116,6 +115,20 @@ public enum FetchGroup {
     @Pattern(regexp = RegexSequence.Definition.PRINTABLE_CHARS, message = "The publisher may only contain printable characters")
     private String publisher;
 
+    @Persistent /**Issue #2373, #2737 */
+    @Column(name = "MANUFACTURE")
```

Comment:
We need to make the new columns nullable, otherwise there'll be constraint violations on existing deployments (where the columns will be `NULL` for all existing components and projects).

Just adding `allowsNull = "true"` to the `@Column` annotation will do the trick!

Reply:
made the change, thanks for the feedback!